Bottom: e3bfa18a88aeda7e99070b7d556e2f977dd68e8a
Top:    fdda37bf0616a8fab1d05335654279eefdceb7f6
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-04-06 01:46:41 +0100

Refresh of drop-in-pcb-gl-code-various-mess

---

diff --git a/src/hid/gtk/gtkhid-gl.c b/src/hid/gtk/gtkhid-gl.c
index 2b4c2d3..cf44922 100644
--- a/src/hid/gtk/gtkhid-gl.c
+++ b/src/hid/gtk/gtkhid-gl.c
@@ -623,6 +623,15 @@ ghid_notify_crosshair_change (bool changes_complete)
     ghid_invalidate_all ();
 }
 
+void
+ghid_notify_mark_change (bool changes_complete)
+{
+  /* FIXME: We could just invalidate the bounds of the mark? */
+  /* FIXME: Seems we're getting prodded before the GUI is up! */
+  if (gport->drawing_area)
+    ghid_invalidate_all ();
+}
+
 static void
 draw_right_cross (gint x, gint y)
 {
